fabfile.py

```diff
@@ -102,7 +106,7 @@ def production():
     """
     Select production instance(s)
     """
-    env.hosts = ['cabot.arachnys.com']
```

Comment:
I know that cabot.arachnys.com was; I just don't think the default should change